support/index.md

```diff
@@ -3,6 +3,7 @@ layout: default
 title: Support
 category: nav
 weight: 2
+redirect_from: /donate
```

Comment:
Ideally this would be in the same format as  in the other pull request https://github.com/FarsetLabs/farsetlabs.github.io/pull/214, but the codebase is generally in need of a large refactor so I'm not going to be picky.